tests/indicators/algorithms/relativeStrengthIndicatorSpec.js

```diff
@@ -77,14 +77,14 @@
 
             it('should calculate correctly with a windowSize of 2', function() {
                 rsi.windowSize(2);
-                expect(rsi(data)).toEqual([50, 50]);
+                expect(rsi(data)).toEqual([undefined, 50, 50]);
             });
 
             it('should calculate correctly with a windowSize of 3', function() {
                 rsi.windowSize(3);
-                expect(rsi(data)).toEqual([22.222222222222214]);
+                expect(rsi(data)).toEqual([undefined, undefined, 22.222222222222214]);
```

Comment:
Yep, that's it. Previously we manipulated the incoming data so we implicitly had correlation between the input and calculated values. Obviously now we no longer have that...